### DISCUSSION TITLE
@tasks.loop, async, for loops

### DISCUSSION THREAD

**Discussion by daniroade** [Rapptz/discord.py#6617] [Q&A]
Hi, I want to make a task loop that checks through a csv of member info, and sends a birthday message if it finds a match. I think that the way I have written it, the for loop is making it unhappy. Any tips on how I can re-jig this to avoid the error shown in the screenshot?

<img width="886" alt="Screenshot 2021-04-03 at 13 00 40" src="https://user-images.githubusercontent.com/81637689/113478106-974b0b80-947e-11eb-9b8b-f710c3d86e69.png">

**Comment by AbstractUmbra** [comment]
In line 59 `i[0]` is giving you a [`KeyError`](https://docs.python.org/3/library/exceptions.html#KeyError). This is not an issue with discord.py however, so please attempt to resolve this yourself.
The docs page linked tells you the reason this error is raised.
